scripts/kconfig/lxdialog/check-lxdialog.sh

```diff
@@ -2,11 +2,25 @@
 # SPDX-License-Identifier: GPL-2.0
 # Check ncurses compatibility
 
+# Android Q doesn't allow us to call pkg-config for some reason, so
+# stub out pkg-config to avoid build system errors. Uff...
+if [ -n "${PRODUCT_SOONG_NAMESPACES}" ] && [ -n "${TARGET_PRODUCT}" ]; then
+pkgconfig()
```

Comment:
Rename this to pkg-config and you don't need the other function and you don't need to rename the function calls in ldconfig.